chainstate/client.go

```diff
@@ -288,3 +179,37 @@ func (c *Client) ActiveProvider() string {
 	}
 	return ProviderNone
 }
+
+func (c *Client) isFeeQuotesEnabled() bool {
+	return c.options.config.feeQuotes
+}
+
+func (c *Client) initActiveProvider(ctx context.Context) error {
+	switch c.ActiveProvider() {
+	case ProviderMinercraft:
+		return c.minercraftInit(ctx)
+	case ProviderBroadcastClient:
+		return c.broadcastClientInit(ctx)
+	default:
+		return errors.New("no active provider found")
+	}
+}
+
+func (c *Client) checkFeeUnit() error {
+	feeUnit := c.options.config.feeUnit
+	switch {
+	case feeUnit == nil:
+		return errors.New("no fee unit found")
+	case feeUnit.IsZero():
```

Comment:
In addition, we should probably also verify if the Fee unit hasn't got 0 as number of bytes.
And actually probably we should have the check here and when loading fees from miners.

Reply:
Insted of `IsZero` I would create a `IsValid` where I could check both satoshis and bytes. 
Additionally, use only Valid fees returned by miners to calculate the lowest fee. What do you think?

Reply:
AFAIU
if satoshis = 0, then despite the bytes, it is mined for free
if bytes = 0, then the fee seem to be invalid fee, because it is not calculable.
that's why I wouldn't replace IsZero with IsValid, but just add IsValid as a separate case.

Reply:
Agreed, I'll do it.